ovs-persistence/src/main/java/org/dcsa/ovs/persistence/entity/Voyage.java

```diff
@@ -0,0 +1,37 @@
+package org.dcsa.ovs.persistence.entity;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import java.util.UUID;
+
+@Data
+@Entity
+@Table(name = "voyage")
+public class Voyage {
+  @Id
+  @GeneratedValue
+  @Column(name = "id", nullable = false)
+  private UUID id;
+
+  @Column(name = "carrier_voyage_number", length = 50, nullable = false)
+  private String carrierVoyageNumber;
+
```

Comment:
```suggestion

@Column(name = "universal_voyage_reference", length = 5)
private String universalVoyageReference;
```

Reply:
I know this will require a SQL change as well. I just remembered that we might need this (it is part of the OVS 3.0.0 API specification). When @douradofrazer / @pimg reads this tomorrow they can clarify if universalVoyageReference should be included in this release or not...

Reply:
yes, we need to include it, also confirmed with Emilio

Reply:
is universalVoyageReference a replacement of carrierVoyageNumber or is it in aadition too?
Also is 5 chars enought for a universal reference?

Reply:
In addition too for now. I think the length is 9 (maybe 10, but that is pending confirmation if we need the last digit).  The format is "SR" + 5 digits + one upper case character.  E.g., SR00000A.

Reply:
> In addition too for now. I think the length is 8-9 (it is pending confirmation whether we need the last digit). The format is "SR" + 5-6 digits + one upper case character. E.g., SR000000A (for the 6 digit variant).

Nvm me, I was thinking of the service, not the voyage number.

Reply:
Yes - the universalVoyageReference is 4 digits (2 digits for the sequence and 2 for the year + a direction (N, W, E or S)
(It is optional for now - in time (some years from now) it will replace the carrierVoyageNumber)

Reply:
..I will correct myself - it is 4 **characters** followed by a direction
Reason for characters is 99 as largest sequence number might in some cases not be enough - so it is allowed to insert alphanumeric characters as well...

Reply:
universalVoyageReference has been added